.gitignore

```diff
@@ -25,3 +25,5 @@ atlassian-ide-plugin.xml
 /nbactions.xml
 *-test-result.html
 rexster.xml
+sc/
```

Comment:
That is where I store convenience scripts which team is probably not interested in, like "build, unzip and run" with various parameters, etc..